resources/i18n/pt.json

```diff
@@ -1145,8 +1145,8 @@
   "language": {
     "English": "English",
     "Korean": "한국어",
-    "Language": "Língua",
-    "OSDefault": "SO padrão",
+    "Language": "Language",
+    "OSDefault": "OS Default",
```

Comment:
Is there any reason to change this part? I don't know about PT. If you intentionally changed this part, you can merge. The other parts are LGTM.